### DISCUSSION TITLE
Can i did't use Google, AWS dependencies?

### DISCUSSION THREAD

**Discussion by tracer8** [outline/outline#2033] [Self Hosting]
Hi,
I'm trying to install outline docker on my Ubuntu server.
But i want to:
- Disable AWS, Google Authentication
- Only install requirement dependencies like NodeJS, Yarn, Postgres DB, Redis on my current local server.
Can i do it?

**Comment by tommoor** [comment]
It is recommended to run the official docker image at `outline/outlinewiki` to make things much easier. Beyond that you can certainly run Postgres, Redis, and a local S3 alternative like minio on your local server without needing any external services.

**Reply by tracer8** [reply]
Oh, so i can't have any option to use local login instead of Google and Slack login?
About storage, i will try to use Minio.

**Reply by tommoor** [reply]
> Oh, so i can't have any option to use local login instead of Google and Slack login?

No, it has never been a feature of this project.

**Comment by tracer8** [comment]
Thank you!
